hcfb-be/backend-server/app/controller/distribution.controller.js

```diff
@@ -0,0 +1,75 @@
+const db = require('../config/db.config.js');
+const Distribution = db.distributions;
+
+exports.create = (req, res) => {
+    //Save to postgre database
+    Distribution.create({
+      "deliveryLocation" : req.body.deliveryLocation,
+      "deliveryDepartureTime" : req.body.deliveryDepartureTime,
+      "deliveryWeights" : req.body.deliveryWeights,
+      "projectedNumberOfClients" : req.body.projectedNumberOfClients
+    }).then(distribution => {
+        res.json(distribution);
+    }).catch(err => {
+        console.log(err);
+        res.status(500).json({msg: "error", details: err});
+    });
+};
+
+// Find a the most recent distribution
+exports.findByLocation = (req, res) => {	
+    Distribution.findAll(
+        { where: {deliveryLocation: req.params.deliveryLocation} }).then(distribution => {
+			res.json(distribution);
```

Comment:
It looks like this endpoint will return an array; empty if `deliveryLocation` is not found, containing one element if it is. It would be a little more idiomatic RESTful to return a single result or a 404 instead.